### PR TITLE
feat(reset): add try-catches and more logs

### PR DESCRIPTION
**Pull request by hulkoba**

### CODE REVIEW

File: jobs/reset.js [hunk 1]
```diff
@@ -82,40 +87,46 @@ module.exports = async function ({ repositoryFullName }) {
     endkey: `${repoDoc._id}:issue:\ufff0`,
     inclusive_end: true
   })
+  const openIssues = issues.rows.filter(issue => issue.doc.state !== 'closed')
+  log.info(`Started closing ${openIssues} open issues`)
```
Comment (espy) [inline jobs/reset.js]: Don’t you want to log the length of the issues Array like before?

Comment (espy) [reply jobs/reset.js]: And maybe put the issues themselves in the log payload.

Comment (espy) [reply jobs/reset.js]: But much nicer with the filter 👍

Comment (hulkoba) [reply jobs/reset.js]: Of course :)

**Pull request merged**